### ISSUE TITLE
Default settings for test suite, user keyword

### ISSUE THREAD

**Issue by sbrugman** [robotframework/RIDE#1540]
It would be a great feature to be able to set default settings for test cases, test suites and user keywords in preferences!

**Comment by yanne** [comment]
Yes it would. However, we are currently in maintenance-only mode, so I cannot promise that this would be implemented any time soon.
